index.bs

```diff
@@ -12089,6 +12097,93 @@ running the algorithm for an {{AudioNode}}, using an <a>input
 buffer</a> and the value(s) of the {{AudioParam}}(s) of this
 {{AudioNode}} as the input for this algorithm.
 
+<h3 id="interruption-handling">Handling an interruption on the {{AudioContext}}</h3>
+
+An <dfn>interruption</dfn> is an event generated by the user agent when it needs
+to halt audio playback for an {{AudioContext}}. For example, The user agent may
+create an interruption when another application requests exclusive access to the
+audio output hardware. This could happen when there is an incoming call from a
+VoIP application.
```

Comment:
> This could happen when there is an incoming call from a
VoIP application.

Not sure if this sentence is necessary. Perhaps it's better to make it non-normative note with actual examples? (e.g. phone call, etc)

Reply:
We have a non-normative note below, so perhaps we can just cut the sentence out.

Reply:
Removed it. Thanks!